src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost/test?serverTimezone=UTC</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">!Unlimited12345</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
don't forget to leave last line empty
review your PR and do it when you see this sign
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/49767049/4ca6411c-3529-44f7-9e2c-882c01d2b498)